prime-router/metadata/HL7/catchall/hl7/segments/OBX/Observation.yml

```diff
@@ -22,22 +22,13 @@ id:
   valueOf: "GeneralUtils.generateResourceId()"
   expressionType: JEXL
 
-code_noCE:
+code_obx:
```

Comment:
nit: I'd drop the ending off leaving the name as `code` since there's only one code element now

Reply:
Dropped: https://github.com/CDCgov/prime-reportstream/pull/14724/commits/65faec0f08072f996bba0d20eed2fae14b5f5cfe#diff-505fc2760909a6523c6a5ca2998f3296e0a6e7f222fc1601af6ad8ee9b3070bfL25